### PR TITLE
fix word "Decretion" to Decrediton

### PR DESCRIPTION
**Pull request by luisantoniocrag**

### CODE REVIEW

File: app/i18n/translations/es.json [hunk 1]
```diff
@@ -347,7 +347,7 @@
   "input.invalidInput": "Este campo es incorrecto",
   "input.requiredInput": "Este campo es obligatorio",
   "invalidRPCVersion.info": "La API de la cartera que está ahora mismo en marcha ({walletRPCVersion}) no es compatible con Decrediton (versión requerida {requiredWalletRPCVersion}).\n\n        Actualice el daemon (dcrd) y la cartera (dcrwallet) para tener la última versión, y vuelva a intentarlo.",
-  "loader.title": "Le damos la bienvenida a la cartera de Decretion",
+  "loader.title": "Le damos la bienvenida a la cartera de Decrediton",
```
Comment (matheusd) [inline app/i18n/translations/es.json]: This can't be changed here. This needs to be changed in our original master sources at transifex, otherwise it gets replaced every time we reapply the translations.

File: app/i18n/docs/en/InfoModals/TicketAutoBuyer.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Automatic Purchase Information
 
-**Balance to Maintain** If your balance is lower than this number, you will not buy tickets. The default of 0 will use all the funds in your account to buy tickets
+**Balance to Maintain** If your balance is lower than this number, you will not buy tickets. The default of 0 will use all the funds in your account to buy tickets.
```
Comment (matheusd) [inline app/i18n/docs/en/InfoModals/TicketAutoBuyer.md]: This should be in a dedicated PR, since it's a different change than the rest of this PR.

File: app/i18n/docs/es/InfoModals/DecredConstitution.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+Decred (/ˈdi:ˈkred/, /dɪˈkred/, dee-cred) es una criptomoneda segura, adaptable y autofinanciada con un sistema de gobierno comunitario integrado en su cadena de bloques. La misión del proyecto es desarrollar tecnología para el beneficio público, con un enfoque principal en la tecnología de criptomonedas. Esta constitución define un conjunto de principios que guían la toma de decisiones de las partes interesadas del proyecto y describe los procesos a través de los cuales se gobierna la cadena de bloques y el Tesoro. Este documento está destinado a gestionar las expectativas de los usuarios potenciales y reales de Decred, para que entiendan más claramente el contrato social entre ellos y el proyecto. Este documento no tiene prioridad sobre las reglas de consenso de la red Decred, en caso de que surja algún conflicto.
```
Comment (matheusd) [inline app/i18n/docs/es/InfoModals/DecredConstitution.md]: I guess it's debatable what the interplay is between versions of the constitution written in different languages. Are all equally valid? Maybe there needs to be a notice on top about the fact that this is a translation of the original document? 

Maybe this is better off discussed in chat/politeia...

**Comment by alexlyp** [comment]
While I appreciate the work being down here.  These translations aren't being correctly done.

We use transifex to supply our various translations.  

Please come discuss with us on the dev channel of chat.matrix.org.  Thanks!
